testing/tests/DevExpress.ui.widgets.editors/textEditorParts/markup.tests.js

```diff
@@ -131,3 +131,191 @@ module('the \'name\' option', () => {
     });
 });
 
+module('basic options changing', {
+    beforeEach: function() {
+        this.element = $('#texteditor').dxTextEditor({});
+        this.input = this.element.find('.' + INPUT_CLASS);
+        this.instance = this.element.dxTextEditor('instance');
+    }
+}, function() {
+    test('value', function(assert) {
+        this.instance.option('value', '123');
+        assert.strictEqual(this.input.val(), '123');
+
+        this.instance.option('value', '321');
+        assert.strictEqual(this.input.val(), '321');
+    });
+
+    test('the "inputAttr" option', function(assert) {
+        let $div1;
+        let $div2;
+
+        try {
+            $div1 = $('<div>', { id: 'testDiv1' }).appendTo($(document.body));
+            $div2 = $('<div>', { id: 'testDiv1' }).appendTo($(document.body));
+
+            const instance1 = $div1.dxTextEditor({ inputAttr: { 'data-test': 'test' } }).dxTextEditor('instance');
+            const instance2 = $div2.dxTextEditor().dxTextEditor('instance');
+            const $input1 = $div1.find('input');
+
+            assert.ok(typeof (instance2.option('inputAttr')) === 'object' && $.isEmptyObject(instance2.option('inputAttr')), 'Option is {} by default');
+            assert.strictEqual(instance1.option('inputAttr')['data-test'], 'test', 'Option sets to the widget on init');
+            assert.strictEqual($input1.attr('data-test'), 'test', 'Option sets to the widget input on init');
+
+            instance1.option('inputAttr', { 'data-test': 'changedValue', 'data-anyattr': 'anyvalue' });
+
+            assert.strictEqual($input1.attr('data-test'), 'changedValue', 'Attr was changed by API');
+            assert.strictEqual($input1.attr('data-anyattr'), 'anyvalue', 'New attr was set by API');
+        } finally {
+            $div1.remove();
```

Comment:
We can not to remove it because fixture clears before every test

Reply:
But we append this element to `document.body`, not to `qunit-fixture`.

Reply:
I didn't refactor the transferred tests, thanks

Reply:
@San4es I've fixed it too